extra/puppet_sign.rb

```diff
@@ -32,8 +32,8 @@ def log(message)
 protocol         = PUPPETCA[:enabled] == true ? 'https' : PUPPETCA[:enabled]
 port             = protocol == 'https' ? SETTINGS[:https_port] : SETTINGS[:http_port]
 fqdn             = Socket.gethostbyname(Socket.gethostname).first
-# e.g. https://hostname.localdomain.com:8443/puppet/ca/autosign
-uri              = URI.parse("#{protocol}://#{fqdn}:#{port}/puppet/ca/autosign")
+# e.g. POST https://hostname.localdomain.com:8443/puppet/ca/verify
+uri              = URI.parse("#{protocol}://#{fqdn}:#{port}/puppet/ca/verify")
```

Comment:
This should be `/puppet/ca/validate`. See https://github.com/theforeman/smart-proxy/blob/develop/modules/puppetca/puppetca_api.rb#L40